### PR TITLE
Add Polkamarkets (POLK) token

### PR DESCRIPTION
**Pull request by rsmarques**
Part of https://github.com/maticnetwork/polygon-token-list/issues/270

### CODE REVIEW

File: src/tokens/polygonTokens.json [hunk 1]
```diff
@@ -20518,5 +20518,23 @@
...
         }
+    },
+    {
+        "chainId": 137,
+        "name": "Polkamarkets",
+        "symbol": "POLK",
+        "decimals": 18,
+        "address": "0x996F19d4b1cE6D5AD72CEaaa53152CEB1B187fD0",
+        "logoURI": "https://raw.githubusercontent.com/Polkamarkets/polkamarkets-web/main/public/badge.svg",
+        "tags": ["pos", "fx", "erc20", "swapable"],
```
Comment (py-zoid) [inline src/tokens/polygonTokens.json:20529]: The token seems to be mapped over fx bridge. Please remove the "pos" tag.

**Review by py-zoid — CHANGES_REQUESTED** [review]

File: src/tokens/polygonTokens.json (hunk 1 above)
Comment (rsmarques) [reply src/tokens/polygonTokens.json]: @py-zoid done 👍

Comment (py-zoid) [reply src/tokens/polygonTokens.json]: Sorry for not pointing earlier, but could you add "rootTunnelAddress" as "0x0cc2caed31490b546c741bd93dbba8ab387f7f2c". [Here](https://github.com/maticnetwork/polygon-token-list/blob/532277fe4f44040a7e548226274593b8b1b17b88/src/tokens/polygonTokens.json#L20458) is an example you can follow.

We'll add documentation for these custom cases as well into the readme file soon, looks like we missed it. Thank you.

Comment (rsmarques) [reply src/tokens/polygonTokens.json]: @py-zoid just added 👍

**Review by py-zoid — APPROVED** [review]
LGTM

**Pull request merged**